_component_design/buttons.md

```diff
@@ -94,22 +65,25 @@ When using a simple button, specify both weight and size.
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
-
-### Spacing and specs
-
-For most desktop use cases, you should use a standard button size.   
-
-Large buttons should be used in mobile designs to accommodate the ideal touch target size. 
-
-Compact buttons should be used when you have severe space constraints, such as inside of a table or in a dashboard widget.
-
-{% endcolumn %}
-
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-
-{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-specs.svg"/>
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Button labels
```

Comment:
this table looks a bit weird still. can we make the header one line? 

maybe we make it look less like an visual support and more like content on the canvas?